index.Rmd

```diff
@@ -202,19 +200,19 @@ download.file(url = 'https://github.com/GeoScripting-WUR/IntroToRaster/releases/
 unzip(zipfile = 'data/tura.zip', exdir = 'data')
 
 # Retrieve the content of the tura sub-directory
-list <- list.files(path = 'data/tura/', full.names = TRUE)
+turaList <- list.files(path = 'data/tura/', full.names = TRUE)
```

Comment:
Here we named the variable as "list" when the function "list.files()" is also being used. To avoid confusion and show good scripting habits, I changed the variable to "turaList".